src/Analyser.php

```diff
@@ -377,6 +379,38 @@ private function isConstOrFunction(string $usedClass): bool
         return defined($usedClass) || function_exists($usedClass);
     }
 
+    /**
+     * It is almost impossible to sneak a native type here without reaching fatal or parse error.
+     * Only very few edgecases are possible (using \array and \callable).
```

Comment:
Theoretically, we might list only those two here, but this way, it is clear in the test that we dont report anything stupid.